config/empty_retrieval_pipeline.yaml

```diff
@@ -18,7 +18,7 @@ components:
     type: haystack.components.embedders.sentence_transformers_text_embedder.SentenceTransformersTextEmbedder
   prompt_builder:
     init_parameters:
-      template: "{% for document in documents %}\n    {{ document.content }}\n{% endfor %}"
+      template: "{% for document in documents %}\nImage: <|image_\nThis image shows: {{ document.content }} \n{% endfor %}"
```

Comment:
What about textual retrieval? is it still going to work?

Reply:
yes, the placeholder is overridden in the image pre-processing done in the generator, not by the prompt builder, since each MM model replaces differently. For example, llava will replace it with <image> , phi 35 will replace it with <|image_{i}|> where i is the image index.
the {{document.content}} will contain the passage's text, so the info will be present in the prompt

Reply:
I see there's [empty_doc_only_retrieval_pipeline.yaml](https://github.com/IntelLabs/fastRAG/blob/improve_multi_modal_architecture/config/empty_doc_only_retrieval_pipeline.yaml) for textual retrieval so that's fine.